syft/pkg/cataloger/javascript/test-fixtures/pkg-lock/package-lock-3.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
nit/question: it seems that there is no license examples in this fixture, is that intentional? or does the V3 not include this?

Reply:
disregard, I understand your PR comment now 🤦

Reply:
I was also thinking it would be nice to have licenses in the test fixture... it's documented on [the packages](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#packages) field:
> Package descriptors have the following fields:
> ...
> * bin, license, engines, dependencies, optionalDependencies: fields from package.json

I'm not sure offhand how to get a file like this generated, though...

Reply:
Yeah, this is the same issue I had. Given this isn't urgent, let me do some digging and see if I can figure out why these keys appear (or not)

Reply:
https://github.com/npm/cli/issues/5532 << Tracking the `license` issue here